tests/integration/conftest.py

```diff
@@ -33,15 +33,21 @@ def get_engine(backend) -> sa.engine.Engine:
         password = os.environ.get("SNOWFLAKE_PASSWORD")
         account = os.environ.get("SNOWFLAKE_ACCOUNT", "")
         connection_string = f"snowflake://{user}:{password}@{account}/datajudge/DBO?warehouse=datajudge&role=accountadmin"
+    elif "bigquery" in backend:
+        gcp_project = os.environ.get("GOOGLE_CLOUD_PROJECT", "scratch-361908")

```

Comment:
@ivergara Do you expect us to still use these identifiers once CI is set up?

Reply:
No, but it doesn't hurt to keep them there as placeholders.

Reply:
ok will leave it then, also because the name sounds generic and anonym